reportwidgets/NewUsers.php

```diff
@@ -0,0 +1,114 @@
+<?php namespace RainLab\User\ReportWidgets;
+
+use Backend\Classes\ReportWidgetBase;
+use Carbon\Carbon;
+use Exception;
+use RainLab\User\Models\User;
+
+class NewUsers extends ReportWidgetBase
+{
+    /**
+     * Renders the widget.
+     */
+    public function render()
+    {
+        try {
+            $this->loadData();
+        } catch (Exception $e) {
+            $this->vars['error'] = $e->getMessage();
+        }
+
+        return $this->makePartial('widget');
+    }
+
+    /**
+     * Define widget properties.
+     * 
+     * @return array
+     */
+    public function defineProperties()
+    {
+        return [
+            'widget_title' => [
+                'default'           => e(trans('rainlab.user::lang.widgets.new_users.widget_title')),
+                'title'             => 'backend::lang.dashboard.widget_title_label',
```

Comment:
Key doesn't exist

Reply:
I found this string in our [google analytics plugin](https://github.com/rainlab/googleanalytics-plugin/blob/master/reportwidgets/Browsers.php#L35), and it looks like we should have access to it
https://github.com/octobercms/october/blob/master/modules/backend/lang/en/lang.php#L75